build-system/global-configs/prod-config.json

```diff
@@ -35,6 +35,7 @@
   "amp-consent": 1,
   "amp-img-native-srcset": 1,
   "amp-story-v1": 1,
+  "amp-carousel-scroll-snap": 10,
```

Comment:
Shouldn't this be 0.1?

Reply:
Oh lol  - then I turned the experiment on for 100% of canary? Just confirmed with Raghu that this should be 0.1 indeed. Changed this to `0.01` to signify a 1% prod promotion.